Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Components/Users/Helpers/SearchTextFilter.cs

```diff
@@ -16,6 +17,12 @@ public static string CleanWildcards(string searchText)
             {
                 return null;
             }
+
+            if (DateTime.TryParse(searchText.TrimEnd('*'), out var _))

```

Comment:
If this is something to be supported, this should NOT be done in this method. as it is not cleaning a wildcard and the intention would not be clear

Reply:
Agreed @mitchelsellers